.github/workflows/update-example-providers.yml

```diff
@@ -0,0 +1,21 @@
+name: update-example-providers
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
+  workflow_dispatch:
+
+permissions:
+  contents: read
+
+jobs:
+  renovate:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4.2.2
+      - name: Self-hosted Renovate
+        uses: renovatebot/github-action@v41.0.5
```

Comment:
Would you mind running this workflow through https://app.stepsecurity.io/secureworkflow and grabbing the SHA1s from there instead of the version tags?